src/scss/_slider-input.scss

```diff
@@ -1,51 +1,104 @@
-$slider-handle-size: 2.2rem;
+$thumb-size: 20px;
```

Comment:
All units should be in `rem`.

This would become `2rem`, `18px` would become `1.8rem` etc etc.

It's a good idea to keep everything the same value so you can do math calculations... When you mix units you can't do that anymore.